commons/util/pom.xml

```diff
@@ -52,20 +52,24 @@
       <artifactId>avro</artifactId>
       <version>${avro.version}</version>
     </dependency>
-
+    <dependency>
```

Comment:
After moving the URIBuilder I had to adjust the dependencies - it required the microcks-model as well as the spring-boot-starter-web dependency to build successfully.